### PR TITLE
Adds a light in the Bar Manager's Room

### PR DESCRIPTION
**Pull request by Alanii**
### Because adding a light is too hard.

:(

### CODE REVIEW

**Comment by seekin4u** [comment]
Conflicts, bro

**Comment by hyperioo** [comment]
You need to use the map merger tools to avoid conflicts.

**Comment by hyperioo** [comment]
@Alanii I think it's the deletion of the end line that causes a conflict, try to open `maps/CEVEris/_CEV_Eris.dmm` in a text editor, put back the empty line at the end and push it to your fork.

File: maps/CEVEris/_CEV_Eris.dmm [hunk 1]
```diff
@@ -13484,4 +13484,3 @@ aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
...
 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 "}
-
```
Comment (hyperioo) [inline maps/CEVEris/_CEV_Eris.dmm:13487]: Put back this line.

**Comment by SpadesSlick** [comment]
Author will not be able to get to their PC for a while, PR is stalled, and closed.

**Pull request closed without merging**